File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    revers_list = list(reversed(numbers))
    # print(numbers)
    # print(revers_list)
    for i in range(len(numbers)):
        for j in range(len(numbers)-i-1):
            answer.append(numbers[i]+revers_list[j])
            
    answer=list(set(answer))
    answer.sort()
            
    return answer

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.cpp
#include <string>
#include <vector>
#include <set>

using namespace std;

vector<int> solution(vector<int> numbers) {
    set<int> set_answer;
    
    for(int i=0;i<numbers.size()-1;++i){
        for(int j=i+1;j<numbers.size();++j){
            set_answer.insert(numbers[i]+numbers[j]);
        }
    }
    
    vector<int> answer(set_answer.begin(), set_answer.end());

    return answer;
}